src/main/java/picard/fingerprint/Fingerprint.java

```diff
@@ -90,25 +90,6 @@ public Fingerprint merge(final Fingerprint other) {
         return this;
     }
 
-    /**
-     * Attempts to filter out haplotypes that may have suspect genotyping by removing haplotypes that reach
-     * a minimum confidence score yet have a significant fraction of observations from a third or fourth allele.
-     */
-    public void filterSuspectSites() {
```

Comment:
Have you checked usages of this in picard-private? Perhaps @yfarjoun could comment on whether this method is likely to be in use / useful?

Reply:
@droazen I can't find any uses of it in picardPrivate or the entirety of public github.  It was last touched in 2016 by Eric.